.circleci/config.yml

```diff
@@ -422,19 +414,16 @@ commands:
               docker push ${GCP_IMAGE}-release:${CIRCLE_SHA1}
               docker push ${GCP_IMAGE}-release:${SHORT_GIT_HASH}
               docker push ${GCP_IMAGE}-release:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM}
-              docker push ${GCP_IMAGE}-release:${START_TIME}
-              docker push ${GCP_IMAGE}-release:${START_TIME}-${CIRCLE_SHA1}
               docker push ${GCP_IMAGE}-release:latest
             fi
 
             ### soon obsolete ###
             if [[ ${CIRCLE_BRANCH} == "testnet" ]];then
-               echo ${DOCKERHUB_PASS} | docker login -u taraxa --password-stdin
-               docker push taraxa/${IMAGE}:${CIRCLE_SHA1}
-               docker push taraxa/${IMAGE}:${VERSION}
```

Comment:
Just making sure you know you also removed this `docker push taraxa/${IMAGE}:${VERSION}`